packages/design-tokens/tokens/animation.json.d.ts

```diff
@@ -1,5 +1,5 @@
 import { DeepMapObjectLeafs, Theme } from "../src/types"
-import { useTheme, ThemeProvider } from "../react"
+import { useTheme, ThemeProvider } from "../src/react"
```

Comment:
These `.d.ts` files should be pointing to the `dist` files (generally they shouldn't even be committed), but we opted not to fix it in this PR.